common/README.md

```diff
@@ -37,10 +37,16 @@ The DNS zones are:
 1. By default you'll use the RtF Organization and Billing Account.
    * You can use a different Organization or Billing Account, e.g. from a GCP Free Trial Account, with `export ORGANIZATION_ID=111111111111` and/or `export BILLING_ID=222222-222222-222222`.
 1. Check that [you have the AWS credentials](../aws#configure-cloud-provider-credentials).
-   * Copy the `credentials` file to the `common/.config/prd/aws` directory.
+   * Be sure that your AWS credentials are in your $HOME/.aws directory.
 1. `cd gpii-infra/common/live/prd`
 1. `rake infra_init`
    * This will create a project called `gpii-common-prd`, with all the resources needed to run Terraform and create all the organization projects.
    * This step must be executed by an user with admin privileges in the organization, because it needs to create IAMs that are able to create projects and associate the billing account to them.
 1. `rake apply_infra`
    * This will create all the projects in the organization. Each project is defined by the content of a directory in `common/live/(stg|prd)/infra`
+
+WARNING: The command `rake destroy_infra` of the GCP part of this project can disable the DNS API driving to an issue where Terraform is unable to refresh the state at the next executions. Avoid the use of `rake destroy_infra`, and only remove the most expensive resources using `rake destroy`.
```

Comment:
This seems like a pain. Assuming we can't fix it now, can you create a ticket and reference that ticket from here?

Reply:
Ticket created: https://issues.gpii.net/browse/GPII-3332